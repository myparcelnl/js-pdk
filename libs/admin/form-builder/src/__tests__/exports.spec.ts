import {describe, expect, it} from 'vitest';
import * as index from '../index';

describe('exports', () => {
  it('exports from index.ts', () => {
    expect(Object.keys(index)).toMatchSnapshot();
  });
});
