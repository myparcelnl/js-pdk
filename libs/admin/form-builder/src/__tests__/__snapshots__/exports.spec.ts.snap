// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`exports > exports from index.ts 1`] = `
[
  "buildAfterUpdate",
  "buildFormStateWatcher",
  "createPropSetter",
  "createValueGetter",
  "createValueSetter",
  "executeCustomOperationHandlers",
  "executeOperation",
  "executeOperations",
  "executeSetValueOperation",
  "parseBuilders",
  "validateIfCondition",
  "validateIfConditions",
  "validateIsTruthy",
]
`;
